src/components/List/List.jsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { getPosts } from '../../selectors/Selectors';
+import Post from '../Post/Post';
 
-import Post from '../../components/Post/Post';
+const List = () => {
 
-export default function List() {
-  const posts = useSelector(getPosts);
+  const post = useSelector(getPosts);
 
-  const postElements = posts
-    .map((post, i) => (
-      <li key={post.index}>
-        <Post {...post}/>
-      </li>
-    ));
+  const postElements = post.map(post => (
```

Comment:
by default .map can send the index as the second parameter. 
https://www.w3schools.com/jsref/jsref_map.asp
```
posts.map((post, i) => (
do stuff. 
))
```